### DISCUSSION TITLE
Why the contact point breaks when an object slides?

### DISCUSSION THREAD

**Discussion by 2-woo-10** [google-deepmind/mujoco#1980] [Asking for Help]
Hello.

I'm a student and I'm going to use Mujoko to get sensor information about situations where objects slide through Mujoko.

Currently, I want to get sensor data on situations where I grab the edge of an object and slide while holding it up.

When enough grip force was not applied to lift the object, the object slipped.

At this time, the contact force and contact point are disconnected when the gripper moves with the object, may I know the reason for this?

The image below is a graph that visualizes the contact force, contact point, and the collected data when I test it.

![Screenshot from 2024-06-30 16-39-54](https://github.com/google-deepmind/mujoco/assets/109202740/4a806597-d8ff-47d6-be47-e0eda4b44f92)
![Screenshot from 2024-06-30 16-40-22](https://github.com/google-deepmind/mujoco/assets/109202740/fa8941d9-d317-4d23-9d90-aba0637067d8)
![Screenshot from 2024-06-30 16-40-37](https://github.com/google-deepmind/mujoco/assets/109202740/f09d3246-adc1-433d-93c2-7a6d680705d7)
![Screenshot from 2024-06-30 16-41-00](https://github.com/google-deepmind/mujoco/assets/109202740/87d8eb48-cf9e-45a4-9c4c-63109efda891)
![Screenshot from 2024-06-30 16-41-33](https://github.com/google-deepmind/mujoco/assets/109202740/6a1f5b69-3aad-43d8-b5f5-4c872f773d8f)

**Comment by yuvaltassa** [comment]
Is this with pyramidal or elliptic cones?

**Comment by 2-woo-10** [comment]
> Is this with pyramidal or elliptic cones?

I am using elliptic cones!!!
